sdk/iot/azure-iot-modelsrepository/azure/iot/modelsrepository/resolver.py

```diff
@@ -0,0 +1,148 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import logging
+import six
+import json
+import abc
+import re
+import os
+from .chainable_exception import ChainableException
+
+logger = logging.getLogger(__name__)
```

Comment:
Planning to add more logging plumbing throughout?  I want to ensure we are applying our [best effort](https://azure.github.io/azure-sdk/python_implementation.html#python-logging-usage) with respect to logging events at the right levels in key operations so our users can quickly diagnose a problem (outside of logging we get for free).

Reply:
I will add judicious and consistent logging once the design is finalized - I want to make sure that logging is equally applied throughout and nothing slips through.